src/LanguageServer/Impl/Implementation/Server.WorkspaceSymbols.cs

```diff
@@ -95,7 +98,7 @@ await GetModuleVariablesAsync(entry as ProjectEntry, opts, @params.query, 50, ca
                 _kind = m.MemberType.ToString().ToLowerInvariant()
             };
 
-            var loc = m.Locations.FirstOrDefault(l => !string.IsNullOrEmpty(l.FilePath));
+            var loc = m.Locations.MaybeEnumerate().FirstOrDefault(l => !string.IsNullOrEmpty(l.FilePath));
```

Comment:
Does this (and the other `MaybeEnumerate()` call below) need `WhereNotNull()` like the change above?

Reply:
Yep. Added.

Reply:
You added `WhereNotNull` to the lines below, but not this line.